pkgs/top-level/php-packages.nix

```diff
@@ -247,9 +247,9 @@ lib.makeScope pkgs.newScope (self: with self; {
 
     memcached = callPackage ../development/php-packages/memcached { };
 
-    mongodb = pkgs.darwin.apple_sdk_11_0.callPackage ../development/php-packages/mongodb {
-      inherit (pkgs.darwin.apple_sdk_11_0.frameworks) Security;
-      inherit (pkgs.darwin.apple_sdk_11_0) Libsystem;
+    mongodb = callPackage ../development/php-packages/mongodb {
+      inherit (pkgs.darwin.apple_sdk_11_0) stdenv;
```

Comment:
Does it work on Linux?

Reply:
yep looks like, it's building here.

Reply:
I'm sure `darwin.apple_sdk_11_0.stdenv` doesn't evaluate on Linux. So `stdenv` seems pretty unused here.

Reply:
You're right, I added 
![image](https://github.com/NixOS/nixpkgs/assets/252042/f6f8d989-d6fb-49ff-8451-6714495732ce)

and

![image](https://github.com/NixOS/nixpkgs/assets/252042/a5389317-3a43-4381-b36c-b6fc74e1334d)

And now I get an error trying to evaluate it.

I guess I can get rid of it?

Reply:
I think `stdenv = if stdenv.isDarwin then pkgs.darwin.apple_sdk_11_0.stdenv else stdenv` is the most common pattern. A `// lib.optionalAttrs stdenv.isDarwin { inherit (pkgs.darwin.apple_sdk_11_0) stdenv; }` should also do the trick, though.

Reply:
I removed it for now, it seems to build properly without it... let's wait and see the CI.